src/components/Account/Account.css

```
@@ -5,62 +5,62 @@ body {
 .outer-div {
   max-width: 100vw;
 }
-.heading {
-  display: flex;
-  justify-content: space-between;
-  margin-left: 8vw;
-  margin-right: 8vw;
-  margin-top: 1rem;
-  border-bottom: 1px solid black;
-}
+    .heading {
+      display: flex;
+      justify-content: space-between;
+      margin-left: 8vw;
+      margin-right: 8vw;
+      margin-top: 1rem;
```

use shorthand properties here
```suggestion
      margin: 1rem 8vw 0 8vw;
      
```